plugins/religion.py

```
@@ -36,3 +36,381 @@ def koran(inp):  # Koran look-up plugin by Ghetto Wizard
+Weak overcomes strong, soft overcomes hard. Everyone knows this, no one attains it. 
+Therefore the Sage says: Accept a country's filth And become master of its sacred soil. Accepts country's ill fortune And become king under heaven. True words resemble their opposites.""",
+"""Appears great hatred sand hatred will remain. How can this be good? 
+Therefore the Sage Holds the tally But does not judge people. 
+Those who have Te Control the tally. Those who lack Te Collect their due. 
+Heaven has no favourites But endures in good people.""",
+"""Small country, few people - Hundreds of devices, But none are used. People ponder on death And don't travel far. 
+They have carriages and boats, But no one goes on board; Weapons and armour, But no one brandishes them. 
+They use knotted cords for counting. Sweet is their food, Beautiful their clothes, Peaceful their homes, Delightful their customs. 
+Neighboring countries are so close You can hear their chickens and dogs. But people grow old and die Without needing to come and go.""",
+"""Sincere words and not pretty. Pretty words are not sincere. 
+Good people do not quarrel. Quarrelsome people are not good. 
+The wise are not learned. The learned are not wise. 
+The Sage is not acquisitive - Has enough By doing for others, Has even more By giving to others. 
+Heaven's Tao Benefits and does not harm. The Sage's Tao Acts and does not contend."""]
+
+@hook.command('laozi')
+@hook.command
+def tao(inp):
+    ".tao <chapter:line> -- gets line <line> from chapter <chapter> of the Tao Te Ching (Addiss)"
+
+    resp = ""
+
+    if ':' not in inp:
+        return tao.__doc__
+
+    chapter, line = inp.split(':', 1)
+
+    try:
+        chapter = int(chapter)
+        line = int(line)
+    except ValueError:
+        return tao.__doc__
+
+    if chapter <= 81:
+        target_chapter = tao[chapter - 1].split("\n")
+        try:
+            resp = target_chapter[line - 1]
```

Similar to the chapter check, you can fail early here:

```python
if line > len(target_chapter):
    return "Invalid line {}, chapter {} has {} lines.".format(line, chapter, len(target_chapter)
```